docs/foundations/typography/font-family/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Font family
+heading: Typography
+sidenavTitle: Typography
+layout: layouts/pages/has-toc.njk
+permalink: /foundations/typography/font-family/index.html
+tags: 
+  - typography
+subnav:
+  collection: sortedTypography
+  order: 2
+---
+
+
+<link rel="stylesheet"
+      href="/assets/packages/@rhds/elements/elements/rh-table/rh-table-lightdom.css"
+      data-helmet>
+
+<script type="module" data-helmet>
+  import '@rhds/elements/rh-table/rh-table.js';
+  import '@rhds/elements/rh-card/rh-card.js';
+  import '@rhds/elements/rh-cta/rh-cta.js';
+</script>
+
+## Overview
+
+The Red Hat font family includes 3 fonts: Red Hat Display, Text, and Mono. Their design is based on who we are: real people with big ideas creating platforms that power technology innovation.
```

Comment:
Can we link `Red Hat Display`, `Text`, and `Mono` to their respective sections below?

Also, I wonder if we should use the full name despite the redundancy?  `Red Hat Display`, `Red Hat Text`, and `Red Hat Mono`. Thoughts?